packages/client/src/generation/generateClient.ts

```diff
@@ -360,18 +362,16 @@ export async function generateClient(options: GenerateClientOptions): Promise<vo
   }
 
   // copy the necessary engine files needed for the wasm/driver-adapter engine
-  if (generator.previewFeatures.includes('driverAdapters') && noEngine !== true) {
-    const queryEngineWasmFilePath = path.join(runtimeDir, 'query-engine.wasm')
-    const queryEngineWasmTargetPath = path.join(outputDir, 'query-engine.wasm')
-    // some bundlers (eg. webpack) need this file to exist, even if it's empty
-    // this is because they analyze `query-engine.wasm` for references to other
-    // files. It does not matter for us, because we bundle query_engine_bg.js.
-    const dummyQueryEngineBgTargetPath = path.join(outputDir, 'query_engine_bg.js')
```

Comment:
We no longer need a dummy one, since we are not bundling `_bg.js` file anymore and real file will be present at bundling time.